src/app/shared/models/game.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface Game {
+  firstTeam_id: number;
+  secondTeam_id: number;
```

Comment:
be consistent with naming `matchId` and `secondTeam_id` in 2 files, how is it possible?